.github/CODEOWNERS

```
@@ -0,0 +1,4 @@
+/src/main/java/ @woodiertexas @TibiNonEst @JoostMSoftware @peaceheis
```

```suggestion
/src/main/java/ @DebuggyTeam/core 
```
LGTM but imo this should be a team so if things change that can be done through the team instead of the file.